lib/reportportal.rb

```diff
@@ -97,6 +98,29 @@ def send_file(status, path_or_src, label = nil, time = now, mime_type = 'image/p
       end
     end
 
+    # @option options [Hash] options, see ReportPortal::ItemSearchOptions
+    def get_items(filter_options = {})
+      page_size = 100
+      max_pages = 100
+      all_items = []
+      1.step.each do |page_number|
+        raise 'Two many pages' if page_number > max_pages
```

Comment:
typo 
```suggestion
        raise 'Too many pages' if page_number > max_pages
```

Reply:
why do we need this condition?

Reply:
It's in order to prevent endless loops in case of some bug.